app/views/layouts/_lte_navbar.html.erb

```diff
@@ -73,8 +73,8 @@
     <li class="dropdown notifications-menu">
       <a href="#" class="dropdown-toggle" data-toggle="dropdown">
         <i class="fa fa-bell-o"></i>
-        <% if current_organization && current_organization.requests.active.size > 0 %>
-          <span class="label label-warning"><%= current_organization.requests.active.size %></span>
+        <% if current_organization && (current_organization.requests.active.size > 0 || current_organization.partners.awaiting_review.size > 0) %>
```

Comment:
I wonder if we could pull `(current_organization.requests.active.size > 0 || current_organization.partners.awaiting_review.size > 0)` in a single method?

Definitely not a deal breaker :)